shaas.go

```diff
@@ -45,6 +46,7 @@ func main() {
 		log.Println("at=readonly.disabled")
 	}
 
+	http.HandleFunc("/health", handleHealth)
```

Comment:
consider moving to `/>/health` to not conflict with local files

Reply:
i'll keep it as is for now, but ideally we should use setup a config struct, use `envdecode`, and have that endpoint be configurable